src/main/java/com/aws/greengrass/smbridge/StreamDefinition.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.smbridge;
+
+import com.amazonaws.greengrass.streammanager.model.MessageStreamDefinition;
```

Comment:
How do you get this import resolved? I don't see any pom changes.

Reply:
I did make pom changes. Didn't include in this PR to scope down. Will add now.